ortools/linear_solver/linear_solver.cc

```diff
@@ -1511,6 +1511,25 @@ MPConstraint* MPSolver::MakeRowConstraint(const LinearRange& range,
   return constraint;
 }
 
+MPConstraint* MPSolver::MakeIndicatorConstraint(
+    double lb, double ub, const std::string& name,
```

Comment:
1. This method should fail if the solver doesn't support integer variables
2. You can call `MPVariable::integer` to check it's an integer variable on the indicator variable, and possibly test it's bounds (0 & 1).